Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -12369,8 +12382,121 @@ def get_server_config():
     return server_config
 
 
-from DemistoClassApiModule import *     # type:ignore [no-redef]  # noqa:E402
+def content_profiler(func):
+    """
+    A decorator for profiling the execution time and performance of a function.
+
+    This decorator is useful for identifying performance bottlenecks and understanding the time complexity of your code.
```

Comment:
would add a line here about how to display the .prof file after downloading it

Reply:
yeah just a line saying - file can be opened like `pip install snakeviz; snakveiz file.prof`